pkg/vtpm/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apk add --no-cache curl-dev=7.61.1-r3
 
 #Build TPM2-TSS and TPM2-TOOLS
 FROM alpine:3.8 as build
```

Comment:
We have the same issue with the acrn package. Roman says we should specify 3.8.X so that this doesn't change underneath us. But he hasn't tested it yet.

Reply:
Yes, probably this needs to be fixed for fscrypt package as well. Till we find something uniform, can we go ahead with this change? I will raise a separate PR for fscrypt change.

Reply:
FWIW I see some packages use alpine:3.10.3.
Per Roman we need to at least specify the micro version number to avoid being hit by changes.